backend/ml/inputs.py

```diff
@@ -114,29 +110,22 @@ def get_supertrusted_users(self) -> QuerySet[User]:
                 (self.poll_name,),
             )
         )
-        if n_alternatives <= self.SUPERTRUSTED_MIN_ENTITIES_TO_COMPARE:
-            # The number of alternatives is low enough to consider as supertrusted
```

Comment:
I'm not convinced that this "if" and the code inside are useful. I would have probably also removed the notion  of `SCALING_CALIBRATION_MIN_ENTITIES_TO_COMPARE` unless we need it for debugging or automated tests (the 100 top contributors already have more than 20 comparisons).

Reply:
It's definitely not required for the poll "videos". This mechanism was introduced for the poll "presidentielles2022" where calibrating the scaling on top contributors would not make sense.

I think it's nice to keep this possibility for potential future polls, and for tests purposes as well. On small test cases it gives a clear-cut definition for scaling calibration users (i.e any trusted user who compared all available entities).

Reply:
I'm not sure why it would not make sense, but ok if there is a good reason.
Can you check the other comments ?

Reply:
My point was:  in polls with a small number of entities, we can expect that a large proportion of users will have compared the exact same number of entities. So the set of 100 scaling calibration users is not properly defined, and that's why at the time we opted for selecting all contributors who compared the entire set of entities.